native/package.json

```diff
@@ -68,10 +68,11 @@
     "build-configs": "0.0.1",
     "events": "^3.2.0",
     "htmlparser2": "^7.2.0",
-    "i18next": "^20.6.1",
+    "i18next": "^21.10.0",
```

Comment:
I18next v21 was a major breaking release, I think we should upgrade it in a separate ticket (there is already one, see https://issues.tuerantuer.org/browse/IGAPP-1060.